### PR TITLE
AdapterAbstract: bind() generates an event dispatcher when not provided

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
         "php": ">=5.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.6"
+        "phpunit/phpunit": "~4.6",
+        "vectorface/dunit": "~2.1"
```
Comment (bryanlatten) [inline composer.json]: allows you to run `./vendor/bin/dunit -i` to test against hhvm :)

**Comment by mlebrun** [comment]
:+1:

File: src/Adapters/AdapterAbstract.php [hunk 2]
```diff
@@ -30,6 +31,19 @@ public function __construct( EventDispatcherInterface $event_dispatcher = null )
   /**
    * {@inheritDoc}
    */
+  abstract public function addServer( $host, $port );
```
Comment (bryanlatten) [inline src/Adapters/AdapterAbstract.php]: required to be defined to pass hhvm checks

Comment (ingluisjimenez) [inline src/Adapters/AdapterAbstract.php]: why these 2 guys came back?

**Comment by ingluisjimenez** [comment]
:+1:

**Pull request merged**